src/eredis_cluster_monitor.erl

```diff
@@ -81,7 +81,9 @@ get_pool_by_slot(Name, Slot) ->
 
 -spec reload_slots_map(State::#state{}) -> NewState::#state{}.
 reload_slots_map(State = #state{pool_name = PoolName}) ->
-    NewState = case get_cluster_slots(State#state.init_nodes, State) of
+    NewState = case get_cluster_slots(State#state.init_nodes, State, 0) of
+        {error, Reason} ->
+            {error, Reason};
```

Comment:
Does this place directly return `{error, Reason}` if there is an unsafe type? I see that its spec requires the return type to be a NewState. see: https://github.com/emqx/eredis_cluster/blob/728430dcc21caf7526dacde6ad9dd6e669a92d2c/src/eredis_cluster_monitor.erl#L224

Reply:
My bad, now this returns NewState with an empty init_nodes.